<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionnew_file"/>
    <addaction name="actionopen_file"/>
    <addaction name="separator"/>
    <addaction name="actionsave"/>
    <addaction name="actionsaveother"/>
    <addaction name="separator"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <widget class="QMenu" name="menu_X">
     <property name="title">
      <string>编码格式(&amp;X)</string>
     </property>
     <addaction name="actionUTF_8"/>
     <addaction name="actionUnicode"/>
    </widget>
    <addaction name="actionback"/>
    <addaction name="actionrestore"/>
    <addaction name="separator"/>
    <addaction name="actionScrapbooking"/>
    <addaction name="actioncopy"/>
    <addaction name="actionpaster"/>
    <addaction name="separator"/>
    <addaction name="menu_X"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
    <addaction name="actionclose"/>
    <addaction name="actioncloseall"/>
    <addaction name="actionTile"/>
    <addaction name="actionCascading"/>
    <addaction name="actionnext"/>
    <addaction name="actionprev"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionabout"/>
    <addaction name="actionabout_qt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionnew_file"/>
   <addaction name="actionopen_file"/>
   <addaction name="separator"/>
   <addaction name="actionsave"/>
   <addaction name="actionsaveother"/>
   <addaction name="separator"/>
   <addaction name="actionback"/>
   <addaction name="actionrestore"/>
   <addaction name="separator"/>
   <addaction name="actionScrapbooking"/>
   <addaction name="actioncopy"/>
   <addaction name="actionpaster"/>
   <addaction name="actionexit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionnew_file">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/newifile.ico</normaloff>:/image/newifile.ico</iconset>
   </property>
   <property name="text">
    <string>新建文件(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionopen_file">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/open.png</normaloff>:/image/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/save.ico</normaloff>:/image/save.ico</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionsaveother">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/save as.ico</normaloff>:/image/save as.ico</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/exit.ico</normaloff>:/image/exit.ico</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionback">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/Cancel.ico</normaloff>:/image/Cancel.ico</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionrestore">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/restore.ico</normaloff>:/image/restore.ico</iconset>
   </property>
   <property name="text">
    <string>恢复(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionScrapbooking">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/cut.ico</normaloff>:/image/cut.ico</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>剪贴(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/copy.ico</normaloff>:/image/copy.ico</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionpaster">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/paste.ico</normaloff>:/image/paste.ico</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUTF_8">
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="actionUnicode">
   <property name="text">
    <string>GB2312</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actioncloseall">
   <property name="text">
    <string>关闭所有</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>平铺</string>
   </property>
  </action>
  <action name="actionCascading">
   <property name="text">
    <string>层叠</string>
   </property>
  </action>
  <action name="actionnext">
   <property name="text">
    <string>下一个</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+Right</string>
   </property>
  </action>
  <action name="actionprev">
   <property name="text">
    <string>上一个</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+Left</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionabout_qt">
   <property name="text">
    <string>关于QT</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
